cluster-up/cluster/k8s-1.14.2-kind-sriov/manifests/sriovdp-daemonset.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sriov-device-plugin
+  namespace: kube-system
+
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: kube-sriov-device-plugin-amd64
+  namespace: kube-system
+  labels:
+    tier: node
+    app: sriovdp
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: sriovdp
+    spec:
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+        - key: node-role.kubernetes.io/master
+          operator: Exists
+          effect: NoSchedule
+      serviceAccountName: sriov-device-plugin
+      containers:
+        - name: kube-sriovdp
+          image: nfvpe/sriov-device-plugin:latest
```

Comment:
Can we pin it to a specific version (latest is v3.0.0 on nfvpe https://hub.docker.com/r/nfvpe/sriov-device-plugin/tags)?

Reply:
Will need https://github.com/intel/sriov-network-device-plugin/commit/93c15d33ba4c3897985875272440bdc12d759641 to land into an image, it fixes an issue I opened since it caused the dp to crash when not all the interfaces were on the same ns

Reply:
Cool!